var/spack/repos/builtin/packages/py-setuptools/package.py

```diff
@@ -30,14 +30,15 @@ class PySetuptools(PythonPackage):
        upgrading, installing, and uninstalling Python packages."""
 
     homepage = "https://pypi.python.org/pypi/setuptools"
-    url      = "https://pypi.python.org/packages/source/s/setuptools/setuptools-11.3.tar.gz"
+    url      = "https://pypi.io/packages/source/s/setuptools/setuptools-25.2.0.tar.gz"
 
-    version('25.2.0', 'a0dbb65889c46214c691f6c516cf959c',
-            url="https://pypi.python.org/packages/9f/32/81c324675725d78e7f6da777483a3453611a427db0145dfb878940469692/setuptools-25.2.0.tar.gz")
+    version('34.2.0', '41b630da4ea6cfa5894d9eb3142922be',
+            url="https://pypi.io/packages/source/s/setuptools/setuptools-34.2.0.zip")
```

Comment:
In an interesting turn of events, the latest version of `py-setuptools` now depends on `py-packaging`, `py-six`, and `py-appdirs`. The latter 2 aren't in Spack yet, but the first definitely depends on `py-setuptools`. Not sure what the best course of action here is...